cmd/bootkube/recover.go

```
@@ -66,6 +70,22 @@ func runCmdRecover(cmd *cobra.Command, args []string) error {
 			return err
 		}
 		backend = recovery.NewEtcdBackend(etcdClient, recoverOpts.etcdPrefix)
+	case recoverOpts.etcdBackupFile != "":
+		bootkube.UserOutput("Attempting recovery using etcd backup file at %q...\n", recoverOpts.etcdBackupFile)
+
+		err := recovery.StartRecoveryEtcd(recoverOpts.podManifestPath, recoverOpts.etcdBackupFile)
+		if err != nil {
+			return err
+		}
+		defer recovery.CleanRecoveryEtcd(recoverOpts.podManifestPath)
```

nit: ignoring error returned by function.